plugins/dac_data_manager.c

```diff
@@ -1243,7 +1243,7 @@ static char *build_default_channel_name_from_index(guint ch_index)
 	guint tone_index = (ch_index % 2) + 1; /* There are always 2 tones (I/Q). Count starts at 1 */
 	const char i_q_type = (ch_index & 0x02) ? Q_CHANNEL : I_CHANNEL; /* First two indexes are I, next two are Q and so on*/
 
-	return g_strdup_printf("TX%i_%c_F%i", tx_index, i_q_type, tone_index);
+	return g_strdup_printf("TX%u_%c_F%u", tx_index, i_q_type, tone_index);
```

Comment:
this looks like it's a fix commit for a change that was introduced in this PR
maybe move this change in that function
otherwise it's just noise